### ISSUE TITLE
keycloak+opendj

### ISSUE THREAD

**Issue by miroslavcapkovic** [keycloak/keycloak#22351]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

ldap

### Describe the bug

keycloak witth opendj (4.5.4-)  as user federation provider has a lot of problems when TLS in its federation ldap provider is configured

Symptoms:
- searching users in admin console stuck - randomly works and not works, but generally time outs
- authentication of user in browser flow stuck after logon on kc login page - randomly works and not works
- in log file cancellation of active tasks and transaction appears (e.g. ARJUNA012077 and many other)  and there is 'TLS' in stack trace

It seems that there is a problem with TLS implementation in opendj  (latest 4.5.5 opendj looks have something solved in this area - but not i ve not tested it at  this moment )

Workaround: use LDAPS  instead of TLS, or/and test 4.5.5,.

In summary this is probably not bug of KC, but can help another users using opendj.

### Version

20.0.0-22.0.1

### Expected behavior

KC  should be able search cross users either ldap provider is timeouting ... 

### Actual behavior

Not able to search or list users, users can not auth (but this is expected because ldap does not responds). Interesting is that import of users works correctly and with good performance without timeout, but auth against LDAP not).

### How to Reproduce?

setup opendj with TLS 

### Anything else?

no

**Comment by ahus1** [comment]
@miroslavcapkovic - I'm triaging your bug report. It would be great if you could paste here:

* The exceptions you see in the console, as they contain stack traces which help to analyze this
* Screenshots as the reveal more of your configuration
* It is currently unclear when you used LDAP with STARTTLS, and when you use a LDAPS (LDAP URL starting with ldaps://)

Thanks!

**Comment by ahus1** [comment]
Additional information is still missing. 

I'm closing the issue for now - please add more comments and this issue might then be reopened.
